human_analysis/nbconverted/2_identify_generic_genes_pathways.py

```diff
@@ -242,6 +242,17 @@
     gene_summary_file, sep='\t')
 
 
+# In[18]:
+
+
+# Test reproducibility
```

Comment:
Do you want to keep this here, or was this just a one-time test? Or should this be in the version of the notebook in the `tests` directory?

Reply:
This was a one time test that really should be in the test directory, you're right